docs/core-concepts/external-routes.md

```diff
@@ -0,0 +1,11 @@
+# External Routes
+You can also define external routes. External routes are just like normal routes except they must be provided a `host` and they can only have the `name`, `host`, `path`, `query`, and `hash` properties. An external route's host can be provided by a parent. When navigating to an external route, the router will simply push the url to the browser's history.
```

Comment:
"With Kitbag Router" is pretty redundant since we're on the kitbag router docs. The "greatest parent" piece is also a little confusing IMO. How about this version? 

```suggestion
External routes allow you to define routes for things outside of your application. External routes have all the same functionality of normal routes, including [params](/core-concepts/params). When navigating to an external route, the router will simply push the url to the browsers history. 

When defining an external route a `host` must be provided, or it can be inherited from a `parent`.
```

Reply:
I'm good with your version. I mostly didn't like a sentence that starts with "You" but I do kinda like using "Kitbag Router" throughout. Yes they're in our docs but the name might be new to them. The more we can drill it the better maybe.

Reply:
Agreed that starting with "you" is something we should avoid.